components/clarinet-cli/src/frontend/cli.rs

```diff
@@ -60,8 +60,12 @@ macro_rules! pluralize {
 #[cfg(feature = "telemetry")]
 use super::telemetry::{telemetry_report_event, DeveloperUsageDigest, DeveloperUsageEvent};
 
+/// For Clarinet documentation, refer to https://docs.hiro.so/clarinet/introduction.
```

Comment:
It would be more standard if the first line (before the empty line) is just a brief, 1-line (<80 characters) description of clarinet, then these two sentences go below.